src/Pyz/Zed/Application/ApplicationDependencyProvider.php

```diff
@@ -97,7 +98,7 @@ protected function getServiceProvider(Container $container)
             new WebProfilerServiceProvider(),
             new ZedHstsServiceProvider(),
             new FormFactoryServiceProvider(),
-            new TwigCurrencyServiceProvider(),
+            new TwigCurrencyServiceProvider()
```

Comment:
Please keep the diff low, this must be reverted, Please adjust your IDE accordingly (settings of trailing multi line comma).